services/management/ia/templates_service.go

```diff
@@ -371,24 +371,55 @@ func (s *TemplatesService) ListTemplates(ctx context.Context, req *iav1beta1.Lis
 		return nil, status.Errorf(codes.FailedPrecondition, "%v.", services.ErrAlertingDisabled)
 	}
 
+	pageIndex := 0
+	pageSize := defaultPageSize
+	if req.PageParams != nil {
+		pageIndex = int(req.PageParams.Index)
+		pageSize = int(req.PageParams.PageSize)
+	}
+
+	if pageSize <= 0 || pageIndex < 0 {
+		return nil, status.Errorf(codes.InvalidArgument, "Page size (%d) should be positive number and "+
+			"page index (%d) should be non-negative number", req.PageParams.PageSize, req.PageParams.Index)
+	}
```

Comment:
not sure this will be useful, the FE should now the size of the page to show the page links on the UI.

Reply:
This API can be used not only by FE

Reply:
So let's modify also alert rules paging?
And again it's breaking change, but at least API is still in beta.

Reply:
Yep, let's modify

But why it's breaking change? It shouldn't affect UI as it's always send pageSize as I understand

Reply:
I mean now alert templates. Because right now FE doesn't send anything about pages.
So If I merge this PR without FE ticket. FE will get just first 25 templates.